g4py.sh

```diff
@@ -0,0 +1,51 @@
+package: G4PY
+version: "%(tag_basename)s%(defaults_upper)s"
+source: https://github.com/alisw/geant4
+tag: v4.10.01.p03
+requires:
+  - GEANT4
+  - Python-modules
+  - ROOT
+  - boost
+  - XercesC
+  - opengl
+build_requires:
+  - CMake
+  - "Xcode:(osx.*)"
+env:
+  G4PYINSTALL: "$G4PY_ROOT"
+---
+#!/bin/bash -e
+cp -a $SOURCEDIR/environments/g4py/* $INSTALLROOT
```

Comment:
Like I already mentioned, this would just install G4PY to `$BUILDDIR`, as it installs itself into the source directory. I spend roughly 3 days on this problem and I have not found a better solution, that works with an unmodified source.

I will test your suggestion and notify you of the results.